src/store/reducers.js

```diff
@@ -0,0 +1,7 @@
+import {combineReducers} from 'redux';
```

Comment:
I think better place this one in to `modules` to because all our reducers will be there